server/util/clickhouse/schema/schema.go

```diff
@@ -179,8 +179,19 @@ type Execution struct {
 	DiskWriteOperations int64
 
 	// Task sizing
-	EstimatedMemoryBytes int64
-	EstimatedMilliCPU    int64
+	EstimatedMemoryBytes   int64
+	EstimatedMilliCPU      int64
+	EstimatedFreeDiskBytes int64
+	RequestedComputeUnits  float64
+	RequestedMemoryBytes   int64
+	RequestedMilliCPU      int64
+	RequestedFreeDiskBytes int64
+	MeasuredMemoryBytes    int64
+	MeasuredMilliCPU       int64
+	MeasuredFreeDiskBytes  int64
+	PredictedMemoryBytes   int64
```

Comment:
I feel like these names might be somewhat confusing (was the measurement from the execution of this task, or is this the measurement from a previous execution). wdyt about naming these with `LastMeasured` or `PreviousMeasured` as the prefix?

Reply:
yep, I used PreviousMeasured